common/src/main/scala/Configuration.scala

```diff
@@ -201,13 +201,15 @@ case class Configuration(
   def calculateInterestOnLatePayment(
     amount: BigDecimal,
     taxYear: TaxYear,
+    remission: Option[Interval[LocalDate]],
```

Comment:
isn't the calculator supposed to accept a list of remission periods? I think at least one of them does in the access application anyways